docs/tutorial/rust/src/getting_started.md

```diff
@@ -18,7 +20,7 @@ Then we edit `Cargo.toml` to add the slint dependency:
 edition = "2021"
 
 [dependencies]
-slint = "0.2.0"
+slint = "0.2.1"
```

Comment:
This isn't strictly needed, since `0.2.1` will automatically be used with when specifying `0.2.0`.